taglib/mpeg/id3v2/id3v2tag.cpp

```diff
@@ -197,7 +197,7 @@ String ID3v2::Tag::genre() const
   // appended to the genre string.  Multiple fields will be appended as the
   // string is built.
 
-  TextIdentificationFrame *f = static_cast<TextIdentificationFrame *>(
+  TextIdentificationFrame *f = dynamic_cast<TextIdentificationFrame *>(
```

Comment:
If we touch this, it could be optimized a bit. `d->frameListMap["TCON"]` is called three times and the same `dynamic_cast` is made above.

Reply:
I think this is what you meant.